### ISSUE TITLE
[GUI] Display wallet creation date at installer and in settings

### ISSUE THREAD

**Issue by kloaec** [wizardsardine/liana#805]
Core know the wallet creation date, a user should:
- write it down, probably at the same time as they backup their descriptor
- be able to find it on the settings too.

**Comment by darosior** [comment]
Yeah it would be nice to know the wallet creation date in the "wallet" section of the settings in the GUI.

We shouldn't get it from bitcoind though but from our own database: https://github.com/wizardsardine/liana/blob/d5b7a3a7de8fb36b5995bc25fc5171e425a22d8b/src/database/sqlite/schema.rs#L21-L31

An approach to this would be to:
1. Include this field in the `getinfo` RPC command
2. Display this field in the "wallet" section of the settings in the GUI by calling `getinfo`.

**Comment by pythcoiner** [comment]
I can tackle that, but not sure I can deliver for v4

**Comment by darosior** [comment]
Related to #620.
